Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -597,7 +588,7 @@ public final class ManifestLoader: ManifestLoaderProtocol {
                         let compilerResult : ProcessResult
                         do {
                             compilerResult = try result.get()
-                            evaluationResult.compilerOutput = try (compilerResult.utf8Output() + compilerResult.utf8stderrOutput()).spm_chuzzle()
+                            evaluationResult.compilerOutput = try (compilerResult.utf8Output() + compilerResult.utf8stderrOutput()).spm_chuzzle()?.replacingOccurrences(of: manifestPath.pathString, with: originalPath.pathString)
```

Comment:
Changed this to preserve the path of the original manifest file which seems more helpful than showing the path to a temporary file that is being automatically deleted.

Reply:
seems nice, but maybe move up to were we do the path switcheru so the logic is concentrated in one place and we dont have to pass the the "original path" everywhere